yt_dlp/extractor/odkmedia.py

```diff
@@ -0,0 +1,177 @@
+    _TESTS = [{
+        'url': 'https://www.ondemandchina.com/en/watch/together-against-covid-19/ep-1',
+        'info_dict': {
+            'id': '264394',
+            'ext': 'mp4',
+            'duration': 3256.88,
+            'title': 'EP 1 The Calling',
+            'alt_title': '第1集 令出如山',
+            'thumbnail': 'https://d2y2efdi5wgkcl.cloudfront.net/fit-in/256x256/media-io/2020/9/11/image.d9816e81.jpg',
+            'description': '疫情嚴峻，黨政軍民學、東西南北中協同應考',
+            'tags': ['Social Humanities', 'Documentary', 'Medical', 'Social'],
+        }
+    }]
+
+    _QUERY = """query Episode($programSlug: String!, $episodeNumber: Int!, $kind: String!, $part: Int) {
+        episode(
+            programSlug: $programSlug
+            episodeNumber: $episodeNumber
+            kind: $kind
+            part: $part
+        ) {
+            ...EpisodeDetail
+            __typename
+            }
+        }
+
+        fragment EpisodeDetail on Episode {
+            id
+            episodeNumber
+            part
+            kind
+            link
+            nextEpisode {
+                ...NextEpisodeDetail
+                __typename
+            }
+            title
+            titleEn
+            titleKo
+            titleZhHans
+            titleZhHant
+            synopsis
+            synopsisEn
+            synopsisKo
+            synopsisZhHans
+            synopsisZhHant
+            seoSynopsis
+            seoSynopsisEn
+            seoSynopsisKo
+            seoSynopsisZhHans
+            seoSynopsisZhHant
+            seoKeyword
+            seoKeywordEn
+            seoKeywordKo
+            seoKeywordZhHans
+            seoKeywordZhHant
+            seoTitle
+            seoTitleEn
+            seoTitleKo
+            seoTitleZhHans
+            seoTitleZhHant
+            videoCcLanguages
+            videoDuration
+            releaseDate
+            people {
+                ...PeopleItem
+                __typename
+            }
+            images {
+                thumbnail
+                __typename
+            }
+            ageRating {
+                isBlock
+                __typename
+            }
+            __typename
+            }
+
+            fragment NextEpisodeDetail on Episode {
+                program {
+                    slug
+                    __typename
+                }
+                link
+                kind
+                part
+                episodeNumber
+                images {
+                    thumbnail
+                    __typename
+                }
+                title
+                titleEn
+                titleKo
+                titleZhHans
+                titleZhHant
+                __typename
+            }
+
+            fragment PeopleItem on People {
+                role
+                slug
+                nameEn
+                nameZhHans
+                nameZhHant
+                __typename
+            }
+            """
```

Comment:
Can be simplified

Reply:
How to simplify this ? Adding or deleting some of this parameter can return error 400/500

Reply:
Oh, actually they can be changed by some rule, i will change the query a bit